plugin/Record/RecordManagerReadForm.php

```diff
@@ -322,9 +365,14 @@ public function readFormTaskConfigByFormId(int $formId): array
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlFields = "SELECT *
-			FROM {$prefix}oo_plugin_form_taskconfig
-			WHERE `".esc_sql($this->getIdColumnMain())."` = ".esc_sql($formId)."";
+		$sql = $pWpDb->prepare(
+			"SELECT *
+			FROM %i
+			WHERE %i = %d",
+			$prefix."oo_plugin_form_activityconfig",
+			esc_sql($this->getIdColumnMain()),
+			$formId
+		);
 
 		return $pWpDb->get_row($sqlFields, ARRAY_A) ?? [];
```

Comment:
Habe ich übersehen, das muss auch noch in $sql umbenannt werden. 
```suggestion
		return $pWpDb->get_row($sql, ARRAY_A) ?? [];
```